mode-apis-base/src/alertingModels.ts

```diff
@@ -163,12 +163,47 @@ export const isMetricsThresholdAlertCondition = (obj: unknown): obj is MetricsTh
 
 
 
-/**
- * TODO - Define the attributes required for AlertAction
- */
-export interface AlertAction {
+export interface AlertMessageTemplate {
+    readonly title: string;
+    readonly message: string;
+    readonly recoveryMessage?: string;
+}
+
+export type AlertRecipientType = 'email';
+
+export interface AlertRecipient {
+    recipientType: AlertRecipientType;
 }
 
+export interface AlertRecipientEmail extends AlertRecipient {
+    readonly recipientType: 'email';
```

Comment:
If we use enum for `AlertRecipientType` then change this to
```suggestion
    readonly recipientType: AlertRecipientType.EMAIL;
```

Reply:
fixed